.gitignore

```diff
@@ -6,3 +6,10 @@
 .vscode-upload.json
 .*.sw*
 /node_modules
+
+/vendor/
+<<<<<<< HEAD
```

Comment:
what is this?

Reply:
Me who was unable to fix the merge conflict :woman_facepalming: